### DISCUSSION TITLE
Endpoint not shown in API Portal

### DISCUSSION THREAD

**Discussion by gbarbon** [mia-platform/community#51] [Q&A] [answered]
An endpoint listed in the "endpoint" section is not visible in the API Portal. The endpoint is correctly set to be shown in the API Portal (as shown in the image below). Moreover, I cannot find it even inside the `/configurations/config-maps/swagger-aggregator/config.json` file. Is this possible?

FYI I didn't create the endpoint, I'm only revising old endpoints exposed by the project.

![image](https://user-images.githubusercontent.com/6908044/195373258-f3108d5d-f606-4ff8-9a1e-54d77cb91be0.png)

**Comment by fredmaggiowski** [comment, marked as answer]
Hi, is the OpenAPI Specification for this endpoint exposed by some of the project services? My guess is that there is no service documenting this API since it seems an endpoint service a static website for the _Data Visualization_ marketplace component; am I right?

**Reply by gbarbon** [reply]
You mean that the `Data Visualization` component is not exposing an OpenAPI specification, that's why such endpoint is not shown? It makes sense actually. I've just took a look at the service, and it doesn't seems to me that an OpenAPI spec is present in the service. I even suspect that such an endpoint makes no sense at all (probably created by mistake I guess).

**Reply by fredmaggiowski** [reply]
Correct; as for this:

> I even suspect that such an endpoint makes no sense at all (probably created by mistake I guess).

That depends, if you need the frontend to be served you're obviously needing the endpoint; it is simply incorrect to have it flagged to be _Shown in API Portal_ since it's not an actual API

**Reply by gbarbon** [reply]
That's true. Thanks!
